test/mockito_test.dart

```diff
@@ -204,6 +218,96 @@ void main() {
       when(mock.methodWithNormalArgs(argThat(equals(42)))).thenReturn("42");
       expect(mock.methodWithNormalArgs(43), equals("43"));
     });
+    test("should mock method with typed arg matchers", () {
+      when(mock.methodWithSpecialArgs(
+          typed/*<List<int>>*/(any), typed/*<List<int>>*/(any)))
```

Comment:
Can any of these tests or below remove the type comment like Leaf suggested in the README?